### PR TITLE
Update SB3 Contrib doc (ARS) and W&B integration

### PR DESCRIPTION
**Pull request by araffin**
## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [ ] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [ ] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format` (**required**)
- [ ] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [ ] I have ensured `make pytest` and `make type` both pass. (**required**)
- [ ] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

**Comment by araffin** [comment]
@vwxyzjn could you help reviewing that small PR?

File: docs/guide/integrations.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  config = {
+      "policy_type": "MlpPolicy",
+      "total_timesteps": 25000,
+      "env_name": "CartPole-v1",
+  }
+  run = wandb.init(
+      project="sb3",
+      config=config,
+      sync_tensorboard=True,  # auto-upload sb3's tensorboard metrics
+      # monitor_gym=True,  # auto-upload the videos of agents playing the game
```
Comment (vwxyzjn) [inline docs/guide/integrations.rst:31]: any reason to comment out this line?

**Review by vwxyzjn — APPROVED** [review]
Everything else LGTM

File: docs/guide/integrations.rst (hunk 1 above)
Comment (araffin) [reply docs/guide/integrations.rst:31]: yep, to keep the example minimal (video recording is usually slow and not available for every env)

Comment (araffin) [reply docs/guide/integrations.rst:31]: we link to W&B doc for more complete example

Comment (vwxyzjn) [reply docs/guide/integrations.rst:31]: That makes sense

**Pull request merged**